### DISCUSSION TITLE
Kudos on this work!!

### DISCUSSION THREAD

**Discussion by lukepurcell** [larsbaunwall/Beolyd5#2] [Ideas]
Just wanted to reach out and thank you so much for this work, I’ve been hoping someone with the know-how could do this exact thing!! I am in tech, have a Beomaster 5 and know my way around Linux and pi’s and love tinkering so if I can help in any way, let me know! I have recently designed and built some interface boards that take rca and a trigger signal and convert to rj45 powerlink sockets or the 9 pin din powerlink sockets to power on and send sound to Beolab speakers. These would work well with a hifi berry I think too :)

**Comment by larsbaunwall** [comment]
Oh wauw - that sounds super awesome! I'm definitely up for some collab. And especially on the Linux/Pi stacks, help is very welcome :)

Feel free to ping me on discord `larsbaunwall` 👍

**Comment by thomassteen** [comment]
This is awesome. Been looking for something like this for years :-) Really hope you manage to get it working. 
I will probably not be able to help much on the development, but more than happy to help testing :-)

One idea: would be great if the implementation would support the newer Network Link. Currently my setup at home is that my speakers are connected to my TV (Avant). So, the BeoLyd5 (love the name) would need to connect to the TV through network link in order to get sound out on the speakers. 
My setup was like this when I used the BeoMaster 5.

**Reply by djerik** [reply]
I am kind of in the same situation, but my plan is to make the BeoSound 5 a remote control for my Home Assistant server, and thereby control my network link speakers. Would that work for you too?

**Comment by sawasdee1965** [comment]
This would be great when this project would be a succes. Since  couple of months I am the owner of a Beosound 5 Encore, a wonderful piece of equipment. Hope this unit can have a second (long) life with this project. Unfortunately I am not a technician so can not be able to assist in this project. However will follow up progress as I am very interested in this.
